### PR TITLE
[Go]: migrate to gonum.org/v1 packages

### PR DESCRIPTION
**Pull request by sbinet**
`github.com/gonum/...` imports are deprecated and do not see any new improvements.
migrate to use the maintained `gonum.org/v1/...` imports.

### CODE REVIEW

**Comment by sbinet** [comment]
@btracey @kortschak could you have a look as well?

File: perf.go [hunk 1]
```diff
@@ -40,8 +38,7 @@ func init() {
 	//blas64.Use(cgo.Implementation{})
```
Comment (kortschak) [inline perf.go]: This needs to change. There is not `cgo.Implementation`.

**Comment by johnfgibson** [comment]
Thanks for this. I run the microbenchmark suite on my office desktop computer; I'll test it there in a week when I return from JuliaCon.

File: perf.go (hunk 1 above)
Comment (sbinet) [reply perf.go]: done.

**Pull request closed without merging**